app/Http/View/Composers/PublicAdministrationSelectorComposer.php

```diff
@@ -76,12 +76,12 @@ public function compose(View $view)
                 'name',
             ])->sortBy('name')->toArray();
             $publicAdministrationShowSelector = true;
-            $postRoute = route('admin.publicAdministrations.change');
+            $selectTenantRoute = route('admin.publicAdministrations.select');
         } elseif ($authUser) {
-            $publicAdministrationSelectorArray = $authUser->activePublicAdministrations()->get()
+            $publicAdministrationSelectorArray = $authUser->publicAdministrations()->get()
```

Comment:
Sarà il caso di aggiungere un controllo per non visualizzare nella select le PA in cui l'utente è stato sospeso?

Reply:
Si, direi di modificare la definizione della relazione nel modello in modo da includere solo quelle non sospese usando `wherePivot->`. Poi definirei una relazione `publicAdministrationsWithSuspended` per avere la possibilità di visualizzarle tutte (ad esempio nella pagina di tutte le PA dove si evidenzia anche lo stato. Anche nelle altre definizioni di relazione del modello `User` preferirei avere `wherePivot` invece di `where` perché mi sembra più espressivo.